Rails-Girls-Summer-Of-Code-2017.md

```diff
@@ -0,0 +1,56 @@
+
+
+### Who are we?
+
+<p align="center">
+<img src="https://github.com/276linesofCode/blog-posts/blob/master/IMG_3774.JPG" width="200">
+</p>
+
+Team 276linesofCode is an amalgamation of all things tech and electronic. The Team comprises Shravika Mittal and Brihi Joshi, a duo from New Delhi, India, in their sophomore year of College at IIIT Delhi. We are a dynamic duo that believes in learning and strengthening concepts while on the way to developing something.
+
+Shravika is an Undergraduate majoring in Computer Science and Engineering. It's been hardly a year since she's got her hands dirty with programming, and it's already a vital part of her daily schedule. She brings out dedication and hard work in the team, excelling in whatever she does.
+
+Brihi is majoring in Electronics and Communication Engineering and loves to mix and fiddle with creativity in code. She brings in the caffeine-induced, late night coding sessions along with fresh experiments in the team.
+
+
+### What is [RGSoC](https://railsgirlssummerofcode.org/)?
+
+Rails Girls Summer of Code is a three month long award-winning global fellowship program aimed at bringing more diversity into Open Source.
+
+It is about helping __introduce newcomers__ to the world of programming further expand their knowledge and skills, by contributing to a worthwhile Open Source project. The focus is not on producing highly sophisticated code, but rather participants learning transferable skills from their project work. Apart from inducing coding skills in real world projects, the RGSoC inculcates community building, by introducing its scholars to the mentors who are in control of the actual projects.
+
+The RGSoC team structure goes like this :
+1. The Students - A pair of Students who are enthusiastic to work on a chosen Open Source Project.
+1. A Project Mentor - Usually an active contributor in the Open Source Project that is chosen. He/She is responsible for guiding the students in the project related tasks, features that are to be added and overall development of the project.
+1. The Project Coaches - They are selected by the students during the time of the application and might be local (present in the same location as the students) or remote. They usually help the students in handling technical difficulties throughout the course of the program.
+1. The Project Supervisor - They are the representatives of the RGSoC orga team and guide the rest of the team with the procedures of the program.
+ 
+RGSoC has both the sides - technical as well as social. Apart from working on the project, we even need to get involved in social meets such as a team call with the project supervisor, maintaining daily logs about what was achieved and what challenges we encountered that day, writing blog posts about our experiences  etc.
+
+This year, 20 teams were selected among which, 16 are Sponsored teams (We are one of them :smile:) and 4 are Volunteer teams.
+
+### About our mentor - Kelsey
```

Comment:
Mentor intro might fit better at the end of the Who Are We? section. 

You can use this (or modify if you want):

Kelsey is the team's mentor. She has been at Tessel since its beginning and is a member of the Tessel Project Steering Committee. All of the members of the Steering Committee are very excited to work with our RGSoC team– both to improve internal processes and mentoring capabilities, and to have the benefit of two full-time contributors building on Tessel!